resource.go

```diff
@@ -105,12 +105,13 @@ func (r Resource) Groups() []string {
 // these.
 func (r Resource) Tags() map[string]string {
 	t := map[string]string{}
+	tf_count := regexp.MustCompile(`[#%]`)
 
 	switch r.resourceType {
 	case "aws_instance":
 		for k, v := range r.Attributes() {
 			parts := strings.SplitN(k, ".", 2)
-			if len(parts) == 2 && parts[0] == "tags" && parts[1] != "#" {
+			if len(parts) == 2 && parts[0] == "tags" && !tf_count.MatchString(parts[1]) {
```

Comment:
I think `parts[1] != "#" && parts[1] != "%"` would be clearer, here. Could we also get a comment reminding us why we might encounter either symbol? I'm assuming it's a change between Terraform versions.